src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/__tests__/ArtworkAuctionCreateAlertHeader.jest.tsx

```diff
@@ -180,9 +171,6 @@ describe("ArtworkAuctionCreateAlertHeader", () => {
   })
 
   describe("with the feture flag disabled", () => {
-    beforeAll(() => {
-      ;(useFeatureFlag as jest.Mock).mockImplementation(() => false)
-    })
```

Comment:
In addition to removing these mocks, I think we can remove the `describe` block beginning on L182 entirely since there is no longer a flag. 

I've taken care of this in https://github.com/artsy/force/pull/12905/commits/06307f20043f63ceb1a03ea5c21a3921480243ea since I had to deal with merge conflicts there anyway.

(By the way, the fact that this was passing before _and_ after this PR removed the flag suggests this was a spurious success. Maybe the assertion on L199 was not correct?)